DuckDuckGo/AutofillLoginSettingsListViewController.swift

```diff
@@ -33,6 +33,7 @@ enum AutofillSettingsSource: String {
     case appIconShortcut = "app_icon_shortcut"
     case homeScreenWidget = "home_screen_widget"
     case lockScreenWidget = "lock_screen_widget"
+    case newTabPageShortcut = "new_tab_page_shortcut"
```

Comment:
@amddg44 additional source we've discussed. Used here: https://github.com/duckduckgo/iOS/pull/3104/files#diff-657adca585f395367011eb99bb1806cdc06bcbbc7a5ad508d34d7851c7cf2f44R2044